hack/build-manifests.sh

```diff
@@ -229,7 +229,7 @@ EOM
 # Write HCO CRDs
 (cd ${PROJECT_ROOT}/tools/csv-merger/ && go build)
 hco_crds=${TEMPDIR}/hco.crds.yaml
-${PROJECT_ROOT}/tools/csv-merger/csv-merger  --api-sources=${PROJECT_ROOT}/pkg/apis/... --output-mode=CRDs > $hco_crds
+(cd ${PROJECT_ROOT} && ${PROJECT_ROOT}/tools/csv-merger/csv-merger  --api-sources=${PROJECT_ROOT}/pkg/apis/... --output-mode=CRDs > $hco_crds)
```

Comment:
Note this change.
Otherwise `csv-merger` exits with `cannot import absolute path`